topic_imu: "/camera/imu"
delta_t: 0.005

begin_cov: [0.001,    0.000,      0.000,    0.000,    0.000,    0.000,
                        0.000,     0.001,     0.000,    0.000,    0.000,    0.000,
                        0.000,     0.000,     0.001,    0.000,    0.000,    0.000,
                        0.000,     0.000,     0.000,    0.001,    0.000,    0.000,
                        0.000,     0.000,     0.000,    0.000,    0.001,    0.000,
                        0.000,     0.000,     0.000,    0.000,    0.000,    0.001]
acc_noise: [0.001,    0.000,    0.000,
                         0.000,    0.001,    0.000,
                         0.000,    0.000,    0.001]
